base_layer/wallet/src/output_manager_service/service.rs

```diff
@@ -277,13 +277,13 @@ where
     ) -> Result<(), OutputManagerError>
     {
         let request_key = response.request_key;
-        let response = match response.response {
-            Some(BaseNodeResponseProto::TransactionOutputs(outputs)) => Ok(outputs.outputs),
-            _ => Err(OutputManagerError::InvalidResponseError(format!(
-                "Could not parse response with request key: {}",
-                request_key
-            ))),
-        }?;
+
+        let response: Vec<tari_core::transactions::proto::types::TransactionOutput> = match response.response {
+            Some(BaseNodeResponseProto::TransactionOutputs(outputs)) => outputs.outputs,
+            _ => {
+                return Ok(());
```

Comment:
Great change, however a trace! here indicating that the message was received is probably helpful

Reply:
It will fire for every single received Base Node Response whether they are meant for this Service or not? Is that useful?

Reply:
I guess it shows the Services are up and listening. Will do